modules/op-specifying-pipelines-resource-quota-using-priority-class.adoc

```diff
@@ -167,7 +167,7 @@ Ensure that all tasks in the pipeline belongs to the same priority class.
 .Example: YAML for a pipeline run
 [source,yaml]
 ----
-apiVersion: tekton.dev/v1alpha1
+apiVersion: tekton.dev/v1beta1
```

Comment:
Both old and new version is missing workspace binding
```
spec:
  workspaces:
  - name: local-maven-repo
    emptyDir: {}
```

Reply:
Updated. Thanks.